db/migrate/20200131130342_create_chasers_sent.rb

```diff
@@ -0,0 +1,9 @@
+class CreateChasersSent < ActiveRecord::Migration[6.0]
+  def change
+    create_table :chasers_sents do |t|
+      t.string :type
```

Comment:
`type` is used by Rails for Single Table Inheritance, so this might conflict: https://stackoverflow.com/questions/7134559/rails-use-type-column-without-sti